src/lib.rs

```diff
@@ -339,6 +369,13 @@ impl Mold {
         }
       }
     }
+    for include in &self.data.includes {
+      let path = self.clone_dir.join(include.folder_name());
+      if path.is_dir() {
+        fs::remove_dir_all(&path)?;
+        println!("{:>12} {}     ", "Deleted".red(), path.display());
+      }
+    }
```

Comment:
I'm starting to think `clean_all()` should just drop the whole `clone_dir` directory instead of iterating through everything we're currently aware of. It might have artifacts from previous versions of the moldfile, afterall.